libraries/core-react/src/Search/Search.tsx

```diff
@@ -29,6 +27,11 @@ const {
   },
 } = tokens
 
+type ContainerProps = {
+  isFocused: boolean
+  disabled: boolean
+}
+
 const Container = styled.span`
```

Comment:
This works, but we have used the "[generic](https://styled-components.com/docs/api#using-custom-props)" approach suggested by `styled-components` when applying props to styled components.

Please update so it conforms with the approach we have used in our other components 😅

Reply:
Done. W did not see that convention. Thanks!